glusterd2/volume/metadata.go

```diff
@@ -2,11 +2,13 @@ package volume
 
 const (
 	// BlockHostingAvailableSize is a volume metadata to store available size of hosting volume to create block devices.
-	BlockHostingAvailableSize = "_block-hosting-available-size"
+	BlockHostingAvailableSize = "block-hosting-available-size"
```

Comment:
is this a metadata field set on BHV volume? if yes metadata should start with `_` or else user can delete metadata